grammar.y

```diff
@@ -33,6 +33,8 @@ var keywordsNotAllowed = map[string]struct{}{
   "UNION": {},
 }
 
+var createStmtHasPrimaryKey = false
```

Comment:
We make use of this variable during the parsing to see if a primary table constraint was already seen.